gc_project/gears/views.py

```diff
@@ -125,10 +141,7 @@ def get(self, request):
 
     def post(self, request):
         user = User.objects.get(email=request.user.email)
-        if user.phone != request.POST["frmPhone"]:
-            user.phone = request.POST["frmPhone"]
-            user.save()
-
+        phone = request.POST["frmPhone"]
```

Comment:
Yes, I mentioned this in my note above ("Still need to add on the add gear form as well."). Not sure what I was thinking whenever those lines 128-131 were removed. I will take care of this too.